docs/app/pages/components/components-sections/notifications/badge/badge.component.html

```diff
@@ -0,0 +1,85 @@
+<div class="badge-separator">
+    <button class="btn btn-primary" [uxBadge]="'NEW'" badgeSize="small" badgeOverlap="true">
```

Comment:
`[uxBadge]="'NEW'"` can be changed to `uxBadge="NEW"` to avoid any unnecessary bindings. There are a few others that can be converted to static strings too.

The `badgeOverlap` property should be a binding as `badgeOverlap="true"` is the string `true` whereas `[badgeOverlap]="true"` is the a boolean `true` which is correct in this instance.